app/models/concerns/with_status.rb

```diff
@@ -2,7 +2,7 @@ module WithStatus
   extend ActiveSupport::Concern
 
   included do
-    enum status: [:pending, :running, :passed, :failed]
+    enum status: [:pending, :running, :passed, :failed, :errored, :aborted, :passed_with_warnings]
```

Comment:
Yeah, I think so. We should change to something like `classy_enum`, but that means changing some DB thinks - yeah, awful, I know. Would you open an issue?